### PR TITLE
Milestone 1: Add Animal table and Data 

### PR DESCRIPTION
**Pull request by mirwaisfarahi**
The following changes are added with this PR:

- Create table 'animals'
- Insert data into the table 'animals'
- Wrote the following queries:
- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely
- 10.4kg or 17.3kg)
- Attached are screen-shots of the query results as shown below
1-created-animal-table
10-select-operation
9-select-operation
8-select-operation
7-select-operation
new-select
5-select-operation
4-select-operation
3-select-operation
2-inert-operation
2-insert-operation

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    CREATE TABLE animals(id serial PRIMARY KEY NOT NULL, name VARCHAR(50) NOT NULL, 
+    date_of_birth DATE NOT NULL, escape_attempts INT NOT NULL, neutered BOOLEAN NOT NULL, 
+    weight_kg NUMERIC(5,2) NOT NULL);
 );
```
Comment (cynthiainga) [inline schema.sql]: - [x] Please you can't create a table inside another table, you just need to create one table `animals`. Kindly fix that 🙃 🙃

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```
Comment (cynthiainga) [inline queries.sql]: - [x] Kindly make sure the end date is 2019-31-12 as the question is to list the name of all animals born between 2016 and 2019 so it includes the entire 2019 year, please kindly fix that 😁 😁

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
+SELECT name FROM animals where name LIKE '%Agumon' OR name LIKE '%Pikachu';
```
Comment (cynthiainga) [inline queries.sql]: - [x] Here the question is to list the Date of birth of all animals named either "Agumon" or "Pikachu", you should select the date of birth from animals 😉 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @mirwaisfarahi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [x] Descriptive PR ✔️ 
- [x] Good commit history ✔️ 
- [x] Correct Gitflow ✔️ 

## Required Changes ♻️

- [x] Please kindly update your README file and add the necessary information inside 😉
- [x]  As required in the project you should add the screenshots of your queries results to the pull request description, kindly add it to your pull request 🙃 🙃
- [x] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @mirwaisfarahi  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**